components/datatypes/mobile-application.json

```diff
@@ -0,0 +1,133 @@
+          "enum": ["launch", "install", "upgrade"],
+          "meta:enum": {
+            "launch": "Launch",
+            "install": "Install",
+            "upgrade": "Upgrade"
+          }
+        },
+        "xdm:isClose": {
+          "title": "Is close event",
+          "type": "boolean",
+          "description": "Indicates whether or not this is an application close event and contains details of the previous session."
+        },
+        "xdm:closeType": {
+          "type": "string",
+          "description": "Specifies the close event type. A value of close means the application signaled that it has closed, either by a user action or by the device. Unknown means the application did not signal that it closed, which could indicate an application crash, a system failure, or an improperly configured application.",
+          "enum": ["close", "unknown"],
+          "meta:enum": {
+            "close": "Close",
+            "unknown": "Unknown"
+          }
+        },
+        "xdm:isDailyEngUser": {
+          "title": "Is daily engaged user event",
+          "type": "boolean",
+          "description": "Indicates whether or not this is the first application launch of the day."
+        },
+        "xdm:isMonthlyEngUser": {
+          "title": "Is monthly engaged user event",
+          "type": "boolean",
+          "description": "Indicates whether or not this is the first application launch of the month."
+        },
+        "xdm:sessionLength": {
+          "title": "Session length",
+          "type": "integer",
+          "description": "Reports the number of seconds that a previous application session lasted based on how long the application was open and in the foreground. Included with close event."
+        },
+        "xdm:ignoredSessionLength": {
+          "title": "Ignored session length",
+          "type": "integer",
+          "description": "If the previous session lasts longer than the maxSessionLength, that session length is ignored and recorded here. Included with close event."
+        },
+        "xdm:daysSinceFirstUse": {
+          "title": "Days since first use",
+          "type": "integer",
+          "description": "Number of days since first launch event."
+        },
+        "xdm:daysSinceLastUse": {
+          "title": "Days since last use",
+          "type": "integer",
+          "description": "Number of days since last launch event."
+        },
+        "xdm:daysSinceLastUpgrade": {
+          "title": "Days since last upgrade",
+          "type": "integer",
+          "description": "Number of days since last upgrade event."
+        },
+        "xdm:launchesSinceLastUpgrade": {
+          "title": "Launches since last upgrade",
+          "type": "integer",
+          "description": "Number of launch events since last upgrade event."
+        },
+        "xdm:installDate": {
+          "title": "Install date",
+          "type": "string",
```

Comment:
Please add "format": "date-time"

Reply:
Thanks, I missed this. The format is "date", without time.

Reply:
A `date-time` need not include the time - it is a standard ISO 8601 value.

Reply:
Is `"format": "date"` an invalid format?  I do see it in other schemas (address, person, product, subscription, campaign, etc). I wonder if all date formats are specified as `date-time`, how can consumers of schemas differentiate between dates using RFC3339 `date-time` and dates using RFC3339 `full-date` (date without time)?

Reply:
Yes, it is.  Because we are based on JSON Schema, where there is only `date-time`  (https://json-schema.org/understanding-json-schema/reference/string.html#dates-and-times).   NOTE: while there is support for `date` or `time` only in newer version, we are still based on v6.

Reply:
@kevinlind Since RFC 3339 is a subset of ISO 8601, that's fine to use.   But any valid RFC3339 parser will handle any/all valid dates, times or date-times as defined by that spec.   so not sure what the concern is?!?!

Reply:
This document shows that `date` is a valid XDM format, is that no longer the case? https://github.com/adobe/xdm/blob/master/docs/data_types.md#data-type-definitions.
AEP appears to support it as well when creating a custom mixin from the UI. A field of type Date will set the format to `date` in the generated schema. 

The concern is if a customer wants to use a date format without time, there's no way to know that from the schema if XDM only supports a `date-time` format. XDM producers won't know how to handle the format and will need to default to the full date-time format. Manual producers can possibly look at the field description for a clue to the format, but automated producers, such as some of the code in the mobile SDK, cannot.